internal/controller/standalone_pgadmin/pod.go

```diff
@@ -374,10 +378,39 @@ func startupCommand() []string {
 
 		// configDatabaseURIPath is the path for mounting the database URI connection string
 		configDatabaseURIPathAbsolutePath = configMountPath + "/" + configDatabaseURIPath
+	)
+	var (
+		maxBackupRetentionNumber = "1"
+		// One day in minutes for pgadmin rotation
+		pgAdminRetentionPeriod = "24 * 60"
+		// Daily rotation for gunicorn rotation
+		gunicornRetentionPeriod = "D"
+	)
```

Comment:
Thinking out loud: So it seems like we are defaulting to retaining a day's worth of logs whether we have the OpenTelemetryLogs feature enabled or not? Which looks like it was already the case for pgAdmin, but for gunicorn the behavior is changing from retaining based on size to time?

Reply:
🤔 Yeah -- and actually if we go back far enough, we weren't actually making any changes to the default logging until we added OTel. Which makes me think we should maybe only add these logging configs if OTel is enabled...

Reply:
mmhmm

Reply:
OK, we made some changes about where we're logging, so I couldn't put all the logging stuff in as optional, but made some changes.